apiary.apib

```diff
@@ -16,43 +16,76 @@ current session status.
 
 ### Sign In [PUT]
 
-+ Request (application/json)
++ Request (application/vnd.api+json)
+
+    + Headers
+
+            Accept: application/vnd.api+json
+    
+    + Body
+
+            {
+                "data": {
+                    "attributes": {
+                        "username": "john@example.com",
+                        "password": "secret"
+                    }
+                }
+            }
+
+
++ Response 201 (application/vnd.api+json)
 
         {
-            "username": "john@example.com",
-            "password": "secret"
+            "links": {
+                "self": "https://example.com/session"
+            },        
+            "data": {
+                "id": "sessionid123"
+            }
```

Comment:
So this is where things could get really cool.  

>  I couldn't tell from the specs if the include query is only for GET requests or not, any thoughts?

Not sure. Going to dig into this right now.